README.md

```diff
@@ -0,0 +1,143 @@
+# NixOS on the HoneyComb / ClearFog LX2k
+
+This repo contains minimal viable Nix expressions for NixOS installer images and UEFI
+firmwares for the following machines:
+
+* [HoneyComb LX2K][lx2k-store]
+
+# Prerequisites
+
+Before starting, you should ensure you have:
+
+* Any additional required hardware is properly installed.
+  The default memory speed is 3200mhz. If your memory is slower, you will need to specify a different speed.
+* A Linux machine with Nix installed.
```

Comment:
An ARM one, to be more specific :sweat_smile:

Reply:
I... built on amd x86_64?

Reply:
Using a remote aarch64 builder, I'm guessing?